extensions/MyDashboard/web/js/query.js

```diff
@@ -271,6 +271,11 @@ $(function() {
             }
             YAHOO.bugzilla.bugUserLastVisit.update(bug_ids);
             YAHOO.bugzilla.bugInterest.unmark(bug_ids);
+
+	    var query_select = Y.one('#query');
```

Comment:
the indentation here should be consistent with the rest of the file (and not use tabs)
otherwise this looks good!